defender/README.md

```diff
@@ -47,6 +75,59 @@ the Autotasks to verify that the Autotasks are performing as expected.
 
 ### Use of Jest
 
-This code uses Jest to override the several modules (`axios`, `ethers`, and `defender-relay-client`) to mock their actions and avoid performing interactions
+This code uses Jest to override several modules (`axios`, `ethers`, and `defender-relay-client`) to mock their actions and avoid performing interactions
 with the blockchain or the Internet. This approach allows us to simplify testing by simulating all external data sources and then verifying that Autotasks
-performs the functions we expect.
\ No newline at end of file
+performs the functions we expect.
+
+### Serverless Migration status
+
+NAME - STATUS - (PAUSED)
+
+Production
+
+- cToken Monitor - Ready to deploy
+- Datadog Alerts Heat Map - Error 400
+- Datadog Forta Bot Alerts - Error 400
+- Datadog Forta Detection Bot Health - Error 400
+- Forta cToken - Ready to deploy - Paused
```

Comment:
This one and the Forta Governance one should probably be deleted. Their functionality was re-implemented using Defender components (Sentinels and Autotasks) and the Forta versions have been paused ever since.

Reply:
I've noted that Contract Sentinels superseded them. Since they are still on the production account, I will leave them in GitHub for now.